evap/results/tests/test_views.py

```diff
@@ -335,15 +348,15 @@ class TestResultsSemesterEvaluationDetailViewPrivateEvaluation(WebTest):
     @patch('evap.results.templatetags.results_templatetags.get_grade_color', new=lambda x: (0, 0, 0))
     def test_private_evaluation(self):
         semester = baker.make(Semester)
-        baker.make(UserProfile, email='manager@institution.example.com', groups=[Group.objects.get(name='Manager')])
+        manager = make_manager()
         student = baker.make(UserProfile, email="student@institution.example.com")
         student_external = baker.make(UserProfile, email="student_external@example.com")
         contributor = baker.make(UserProfile, email="contributor@institution.example.com")
         responsible = baker.make(UserProfile, email="responsible@institution.example.com")
         responsible_contributor = baker.make(UserProfile, email="responsible_contributor@institution.example.com")
         test1 = baker.make(UserProfile, email="test1@institution.example.com")
         test2 = baker.make(UserProfile, email="test2@institution.example.com")
-        baker.make(UserProfile, email="random@institution.example.com")
+        uninitialized = baker.make(UserProfile, email="uninitialized@institution.example.com")
```

Comment:
I got some questions about these names:

- `responsible_contributor` is not a responsible, it is an editor
- `test1` and `test2` are more like extra students imo
- Neither `uninitialized` nor `random` make a good naming for a non-participant imo. How about `third_party`, `other` or `non_participant` (maybe with `_user` suffix)?